libminifi/include/core/ClassLoader.h

```diff
@@ -24,10 +24,10 @@
 #include <string>
 #include <map>
 #include <memory>
-#include "utils/StringUtils.h"
+
 #include "core/Core.h"
-#include "io/BufferStream.h"
 #include "ObjectFactory.h"
+#include "range/v3/all.hpp"
```

Comment:
Same here. Interestingly, the difference is still about 0.6 seconds despite using less utilities.

```
#include "range/v3/action/sort.hpp"
#include "range/v3/action/unique.hpp"
```